README.md

```diff
@@ -4,68 +4,48 @@
 
 > An opinionated Python framework for creating reproducible, maintainable and modular data science code.
 
+[![Python version](https://img.shields.io/badge/python-3.7%20%7C%203.8%20%7C%203.9%20%7C%203.10-blue.svg)](https://pypi.org/project/kedro/)
```

Comment:
We'll need to remember to update this when we release `3.11` support.

Reply:
Already done in the PR that adds 3.11.

Reply:
Does that also update this repo?

Reply:
Oops sorry didn't realise this was different repo. 😅

Reply:
Good point, no. I can probably add it in now since I'm guessing that support won't be long in coming and who is studying this repo closely in the next few hours/days for python support -- we could be slightly out of step just to be safe it doesn't get forgotten.